pkg/repository/repository.go

```diff
@@ -697,6 +697,19 @@ func (r *repository) ListModelTriggers(ctx context.Context, pageSize, page int64
 		whereArgs = append(whereArgs, expr)
 	}
 
+	if dbModel.OwnerUID().String() != userUID { // for a runner without ownership, they could only view their own logs
```

Comment:
IMO we should not have any permission check logic in repository, this should be done inside `service` level.

Reply:
fixed